src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -14,15 +16,19 @@
     /** Help information should be shown to the user. */
     private final boolean showHelp;
 
+    /** The application should switch tab. */
+    private final Tab switchTab;
```

Comment:
Perhaps can change this variable name to tab?

Reply:
Good suggestion. But I felt that since the other attributes like showHelp and exit are action words, perhaps "switchTab" might be a better name to accurately represent what CommandResult does? Is there a reason why you think "tab" might be a better name?

Reply:
Good suggestion. But I feel that since the other attributes like showHelp and exit are action words, perhaps "switchTab" might be a better name to accurately represent what CommandResult does? Is there a specific reason why you think "tab" might be a better name?